lib/matplotlib/backends/qt_compat.py

```diff
@@ -180,6 +181,15 @@ def is_pyqt5():
     raise AssertionError("Unexpected QT_API: {}".format(QT_API))
 
 
+# Fixes issues with Big Sur
+# https://bugreports.qt.io/browse/QTBUG-87014, fixed in qt 5.15.2
+if (sys.platform == 'darwin' and
+        LooseVersion(platform.mac_ver()[0]) >= LooseVersion("10.16") and
+        LooseVersion(QtCore.qVersion()) < LooseVersion("5.15.2") and
+        "QT_MAC_WANTS_LAYER" not in os.environ):
```

Comment:
this should be `not os.environ.get("QT_...")` so that having the environment variable set to an *empty* (not "0") value is equivalent to it being unset.

Reply:
Indeed, this is a subtle difference here: it seems that `QT_MAC_WANTS_LAYER` is not set by default and if it is "empty", it implies that it has been set! If the user or something else is setting this variable, then maybe we should leave it to them in order to be as less intrusive as possible?

Reply:
ah, that's unusual but sure, ignore my comment then.